docker/test/upgrade/run.sh

```diff
@@ -185,8 +192,6 @@ tar -chf /test_output/coordination.tar /var/lib/clickhouse/coordination ||:
 
 collect_query_and_trace_logs
 
-check_oom_in_dmesg
```

Comment:
Why not?

Reply:
https://github.com/ClickHouse/ClickHouse/pull/44879#discussion_r1135479784